unstructured/ingest/cli/cmds/weaviate.py

```diff
@@ -0,0 +1,58 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliMixin, DelimitedString
+from unstructured.ingest.interfaces import BaseConfig
+
+CMD_NAME = "weaviate"
+
+
+@dataclass
+class WeaviateCliWriteConfig(BaseConfig, CliMixin):
```

Comment:
To make this clearer, since it's mapping to the `SimpleWeaviateConfig` and the `WeaviateWriteConfig`, should have one that extends the write config: 
```
class WeaviateCliWriteConfig(WeaviateWriteConfig, CliMixin):
...
```
Which simply adds the `class_name` click option and another that maps the connection content:
```
class WeaviateCliConfig(BaseConfig, CliMixin):
...
```

This matches the patten we follow elsewhere. Take a look at how the delta table cli is configured.